test/server_test.py

```diff
@@ -221,19 +241,34 @@ def started_imageio(tmpdir, drop_privileges="true"):
 
         # Wait until server is listening - at this point it already dropped
         # privileges.
-        if drop_privileges:
+        if drop_privileges and ticket is None:
             cfg = config.load(str(conf_dir.join("conf.d", "daemon.conf")))
             with http.ControlClient(cfg) as c:
                 r = c.get("/tickets/no-such-ticket")
                 r.read()
-                assert r.status == 404
+                assert r.status == http_client.NOT_FOUND
 
         yield proc
     finally:
         proc.terminate()
         proc.wait()
 
 
+def test_start_with_ticket(tmpdir, tmp_dirs, tmp_ticket):
+    data = b"a" * 8 * KiB + b"b" * 8 * KiB
+    with started_imageio(tmpdir, drop_privileges="false", ticket=tmp_ticket):
+        conf_dir = tmpdir.join("conf")
+        cfg = config.load(str(conf_dir.join("conf.d", "daemon.conf")))
```

Comment:
Blank line after the steup paragraph and a comment before the check will make the test more clear:
   
    # Load configuration
    code...

    # Test that we can download.
    code...

    # Test that we can upload.
    code...